Building from git master requires DEBUG

This might be known and intentional, but without setting DEBUG, the following errors occur during build. The missing variables are wrapped in `#ifdef DEBUG`

```bash
../src/x11/xf86Wacom.c:706:34: error: ‘pInfo’ undeclared (first use in this function); did you mean ‘AgpInfo’?
  706 |         int n = xf86WaitForInput(pInfo->fd, 0);
      |                                  ^~~~~
      |                                  AgpInfo
../src/x11/xf86Wacom.c:706:34: note: each undeclared identifier is reported only once for each function it appears in
../src/x11/xf86Wacom.c: In function ‘wcmDevSwitchMode’:
../src/x11/xf86Wacom.c:857:37: error: ‘priv’ undeclared (first use in this function)
  857 |         return wcmDevSwitchModeCall(priv, is_absolute) ? Success : XI_BadMode;
      |                                     ^~~~
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -51,6 +51,7 @@ jobs:
           - clang
         meson_options:
           - ''
+          - '-Ddebug-messages=false'
           # clang requires b_lundef=false for b_santize, see
           # https://github.com/mesonbuild/meson/issues/764
           - '-Db_sanitize=address,undefined -Db_lundef=false'

--- a/src/wcmCommon.c
+++ b/src/wcmCommon.c
@@ -874,8 +874,6 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 		wcmAxisSet(&axes, WACOM_AXIS_RING2, ds->abswheel2);
 		v3 = ds->stripx;
 		v4 = ds->stripy;
-		v5 = ds->abswheel;
-		v6 = ds->abswheel2;
 		/* use tx and ty to report stripx and stripy for the DBG below */
 		tx = ds->stripx;
 		ty = ds->stripy;

--- a/src/wcmCommon.c
+++ b/src/wcmCommon.c
@@ -840,7 +840,7 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 	int z = ds->pressure;
 	int tx = ds->tiltx;
 	int ty = ds->tilty;
-	int v3, v4, v5, v6;
+	int v[7] = {0};
 	WacomAxisData axes = {0};
 
 	if (priv->serial && serial != priv->serial)
@@ -872,8 +872,8 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 		wcmAxisSet(&axes, WACOM_AXIS_STRIP_Y, ds->stripy);
 		wcmAxisSet(&axes, WACOM_AXIS_RING, ds->abswheel);
 		wcmAxisSet(&axes, WACOM_AXIS_RING2, ds->abswheel2);
-		v3 = ds->stripx;
-		v4 = ds->stripy;
+		v[3] = ds->stripx;
+		v[4] = ds->stripy;
 		/* use tx and ty to report stripx and stripy for the DBG below */
 		tx = ds->stripx;
 		ty = ds->stripy;
@@ -881,14 +881,14 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 	{
 		wcmAxisSet(&axes, WACOM_AXIS_ROTATION, ds->rotation);
 		wcmAxisSet(&axes, WACOM_AXIS_THROTTLE, ds->throttle);
-		v3 = ds->rotation;
-		v4 = ds->throttle;
+		v[3] = ds->rotation;
+		v[4] = ds->throttle;
 	} else
 	{
 		wcmAxisSet(&axes, WACOM_AXIS_TILT_X, tx);
 		wcmAxisSet(&axes, WACOM_AXIS_TILT_Y, ty);
-		v3 = tx;
-		v4 = ty;
+		v[3] = tx;
+		v[4] = ty;
 	}
 
 
@@ -905,19 +905,19 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 	if (ds->proximity)
 		wcmRotateAndScaleCoordinates(priv, &x, &y);
 
-	v5 = ds->abswheel;
-	v6 = ds->abswheel2;
+	v[5] = ds->abswheel;
+	v[6] = ds->abswheel2;
 	if (IsStylus(priv) && !IsArtPen(ds))
 	{
 		/* Normalize abswheel airbrush data to Art Pen rotation range.
 		 * We do not normalize Art Pen. They are already at the range.
 		 */
-		v5 = ds->abswheel * MAX_ROTATION_RANGE/
+		v[5] = ds->abswheel * MAX_ROTATION_RANGE/
 				(double)MAX_ABS_WHEEL + MIN_ROTATION;
-		wcmAxisSet(&axes, WACOM_AXIS_WHEEL, v5);
+		wcmAxisSet(&axes, WACOM_AXIS_WHEEL, v[5]);
 	} else if (IsStylus(priv) && IsArtPen(ds))
 	{
-		wcmAxisSet(&axes, WACOM_AXIS_WHEEL, v5);
+		wcmAxisSet(&axes, WACOM_AXIS_WHEEL, v[5]);
 	}
 
 	DBG(6, priv, "%s prox=%d\tx=%d"
@@ -925,7 +925,7 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 		"\tserial=%u\tbutton=%s\tbuttons=%d\n",
 		is_absolute(priv) ? "abs" : "rel",
 		ds->proximity,
-		x, y, z, v3, v4, v5, v6, id, serial,
+		x, y, z, v[3], v[4], v[5], v[6], id, serial,
 		is_button ? "true" : "false", ds->buttons);
 
 	/* when entering prox, replace the zeroed-out oldState with a copy of

--- a/src/wcmConfig.c
+++ b/src/wcmConfig.c
@@ -985,7 +985,7 @@ static int wcmInitAxes(WacomDevicePtr priv)
 Bool wcmDevInit(WacomDevicePtr priv)
 {
 	WacomCommonPtr common =	priv->common;
-	int nbaxes, nbbuttons, nbkeys;
+	int nbaxes, nbbuttons;
 
 	/* Detect tablet configuration, if possible */
 	if (priv->common->wcmModel->DetectConfig)
@@ -1002,12 +1002,10 @@ Bool wcmDevInit(WacomDevicePtr priv)
 	/* if more than 3 buttons, offset by the four scroll buttons,
 	 * otherwise, alloc 7 buttons for scroll wheel. */
 	nbbuttons = min(max(nbbuttons + 4, 7), WCM_MAX_BUTTONS);
-	nbkeys = nbbuttons;         /* Same number of keys since any button may be
-	                             * configured as an either mouse button or key */
 
 	DBG(10, priv,
-		"(type %d) %d buttons, %d keys, %d axes\n",
-		priv->type, nbbuttons, nbkeys, nbaxes);
+		"(type %d) %d buttons, %d axes\n",
+		priv->type, nbbuttons, nbaxes);
 
 	if (!wcmInitButtons(priv, nbbuttons))
 	{

--- a/src/x11/xf86Wacom.c
+++ b/src/x11/xf86Wacom.c
@@ -700,9 +700,7 @@ void wcmClose(WacomDevicePtr priv)
 
 static int wcmReady(WacomDevicePtr priv)
 {
-#ifdef DEBUG
 	InputInfoPtr pInfo = priv->frontend;
-#endif
 	int n = xf86WaitForInput(pInfo->fd, 0);
 	if (n < 0) {
 		int saved_errno = errno;
@@ -842,12 +840,10 @@ static int wcmDevSwitchMode(ClientPtr client, DeviceIntPtr dev, int mode)
 {
 	InputInfoPtr pInfo = (InputInfoPtr)dev->public.devicePrivate;
 	Bool is_absolute = TRUE;
-#ifdef DEBUG
 	WacomDevicePtr priv = (WacomDevicePtr)pInfo->private;
 
-	DBG(3, priv, "dev=%p mode=%d\n",
-		(void *)dev, mode);
-#endif
+	DBG(3, priv, "dev=%p mode=%d\n", (void *)dev, mode);
+
 	if (mode != Absolute) {
 		if (mode != Relative)
 			return XI_BadMode;

--- a/src/x11/xf86WacomProperties.c
+++ b/src/x11/xf86WacomProperties.c
@@ -447,6 +447,7 @@ static int wcmSetActionProperty(DeviceIntPtr dev, Atom property,
 
 	rc = wcmCheckActionProperty(priv, property, prop);
 	if (rc != Success) {
+#ifdef DEBUG
 		const char *msg = NULL;
 		switch (rc) {
 			case BadMatch: msg = "BadMatch"; break;
@@ -454,6 +455,7 @@ static int wcmSetActionProperty(DeviceIntPtr dev, Atom property,
 			default: msg = "UNKNOWN"; break;
 		}
 		DBG(3, priv, "Action validation failed with code %d (%s)\n", rc, msg);
+#endif
 		return rc;
 	}
 

